changelog/TotHjZ6OTS6wbCFSoKtoNA.md

```diff
@@ -0,0 +1,3 @@
+audience: general
+level: silent
```

Comment:
What do you think about a patch for developers?

Reply:
Update: we discussed over slack, and decided that the changelog should only affect the release notes, and this PR won't affect the release, so it probably is ok as silent.